### ISSUE TITLE
Missing result of 3rd semester

### ISSUE THREAD

**Issue by Moulick** [Sid-2D/IPU-Result-Parser#6]
The result of the 3rd semester is missing

**Comment by Sid-2D** [comment]
Thanks for raising this issue. Are you from 2014 batch? The problem is I couldn't download your 3rd semester result PDFs from the official website. It showed file not available every time I clicked on it. If you have that PDF please mail it to me and I'll upload it.

**Comment by Moulick** [comment]
Yep, I am from the 2014 Batch, I'll ask around if someone has the file. it's been 3 years now so I have little hope.

**Comment by TdevM** [comment]
Here you go. 
https://goo.gl/UipCXQ

**Comment by Sid-2D** [comment]
Thanks a lot @TdevM ! These are hard to find. I'll keep this issue open in case someone wants post 3rd sem results of other streams.
